modules/create_consortium_release.nf

```diff
@@ -24,8 +25,17 @@ process create_consortium_release {
         /root/cbioportal \
         $release
         """
-    }
-    else {
+    } else if (staging) {
+        """
+        # Fixes renv issue
```

Comment:
Is there more context you can link to about this `renv issue`? I don't know what this would be referring to, and coming across this as I am reading the script leaves me a bit confused.

Reply:
Yah I'm not sure either, the comment was present prior. I'd have to re-run the code as what it was prior (`python root/Genie/...`) to see why that wasn't working for this step and we have to `cd` into the directory first.

Reply:
We have to do this because renv activates when you run Rscript from where the renv.lock file lives. That's why there is the `cd`.

It's almost everywhere in genie-bpc-pipeline